modules/quality_control/readme.md

```diff
@@ -0,0 +1,28 @@
+## Imaging Quality Control Module
+
+#### Purpose
+The purpose of this module is to verify and help ensure imaging data has been uploaded into LORIS and reviwed for Quality Control (QC). It provides a queryable interface for users to gauge the completeness of imaging data collections matched against a given protocol, and the status of QC review processes on them.
+
+Note: this module is not the module in which QC is performed on images, just the module in which QC is *reviewed*.  QC is actually marked on the images in the Imaging Browser module.
+
+#### Intended Users
+This module is used by study administrators, imaging scientists, analysts, coordinators, and clinicians.
+
+#### Scope
+The Imaging QC module allows you to view specific data on uploaded image files, with many additional filtering options. The resulting table has a column for QC status, so you can see whether the upload has passed QC or not. The table provides a link to each file. You can download the table in `.csv` format. 
+
+Note: To QC a scan, the user must click the link leading to the Imaging Browser, and enter the QC details in that module.
+
+#### Permissions
+`Placeholder - waiting for contribution from LL`
```

Comment:
Needs updating before merge.

Reply:
yes, tagged Liza.  if she can't, I'll cover it this week.

Reply:
only one permission is required. the `quality_control` permission gives you access to the module

Reply:
👍  see my next commit